skimage/filters/rank/bilateral.py

```diff
@@ -205,7 +205,7 @@ def sum_bilateral(image, selem, out=None, mask=None, shift_x=False,
 
     See also
     --------
-    skimage.filters.denoise_bilateral for a Gaussian bilateral filter.
+    skimage.filters.gaussian for a Gaussian bilateral filter.
```

Comment:
This is incorrect. `filters.gaussian` is a standard Gaussian filter. See [wikipedia](https://en.wikipedia.org/wiki/Bilateral_filter) for the definition of a bilateral filter.

Probably this is meant to point to `skimage.restoration.denoise_bilateral`?

Reply:
good catch. I've been looking at my computer screen too long today.